core/base/src/TError.cxx

```diff
@@ -111,8 +110,8 @@ ErrorHandlerFunc_t GetErrorHandler()
 
 void ErrorHandler(Int_t level, const char *location, const char *fmt, std::va_list ap)
 {
-   TTHREAD_TLS(Int_t) buf_size(256);
-   TTHREAD_TLS(char*) buf_storage(nullptr);
+   thread_local Int_t buf_size(256);
+   thread_local char *buf_storage(nullptr);
```

Comment:
This used not to work on macOS, not sure if it's still true or not.

Reply:
Seems like in old versions of macOS it didn't even compile.  So if it wasn't working, the PR builder should have caught it.